main.go

```diff
@@ -399,12 +402,24 @@ func main() {
 	if cfg.SplitMessageBytes == 0 {
 		cfg.SplitMessageBytes = 4000
 	}
-
-	bot_tmp, err := tgbotapi.NewBotAPI(cfg.TelegramToken)
+	
+	os.Setenv("HTTP_PROXY", cfg.ProxyLink)
+	fmt.Println("Proxy URL: ", os.Getenv("HTTP_PROXY"))
+	
+	proxyUrl, err := url.Parse(cfg.ProxyLink)
+	if err != nil {
+	    fmt.Println("Bad proxy URL", err)
```

Comment:
Proxy option doesn't seem to be optional, and it should be for 99% of users who don't need this functionality